libs/mrwebrtc-unityplugin/src/PlatformBase.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+// Standard base includes, defines that indicate our current platform, etc.
+
+#include <stddef.h>
+
+
+// Which platform we are on?
+// UNITY_WIN - Windows (regular win32)
+// UNITY_OSX - Mac OS X
+// UNITY_LINUX - Linux
+// UNITY_IPHONE - iOS
+// UNITY_ANDROID - Android
+// UNITY_METRO - WSA or UWP
+// UNITY_WEBGL - WebGL
+#if _MSC_VER
+	#define UNITY_WIN 1
+#elif defined(__APPLE__)
+	#if defined(__arm__) || defined(__arm64__)
+		#define UNITY_IPHONE 1
+	#else
+		#define UNITY_OSX 1
+	#endif
```

Comment:
With Apple moving over to ARM on their OSX devices, this code will define all new Macs as iPhones.

Reply:
What define could be alternatively used?

Reply:
```suggestion
	#if TARGET_OS_TV
		#define UNITY_TVOS 1
	#elif TARGET_OS_IOS
		#define UNITY_IOS 1
	#else
		#define UNITY_OSX 1
	#endif
```
From https://github.com/Unity-Technologies/NativeRenderingPlugin/blob/master/PluginSource/source/PlatformBase.h - I haven't tested it but it looks sensible. Also the same file defines `UNITY_ANDROID` explicitly - why isn't this defined here?